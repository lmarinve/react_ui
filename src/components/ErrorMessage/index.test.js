// Link.react.test.js
import React, {useState} from 'react';
import renderer from 'react-test-renderer';
import { testEnvironment } from '../../../jest.config';
import Index from './index'


test('ErrorMessage.Index Snapshot Test', () => {
    const component = renderer.create(
        <Index />

    );
let tree = component.toJSON();
expect(tree).toMatchSnapshot()
});