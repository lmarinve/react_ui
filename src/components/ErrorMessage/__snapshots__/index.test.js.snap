// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage.Index Snapshot Test 1`] = `
<div
  className="error-message-container"
  style={
    Object {
      "marginBottom": "",
    }
  }
>
  <span>
     
     
  </span>
</div>
`;
